docusaurus.config.js

```diff
@@ -70,98 +69,98 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       // Replace with your project's social card
-      image: 'img/superflows-name-logo.png',
+      image: "img/superflows-name-logo.png",
       themeConfig: {
         colorMode: {
-          defaultMode: 'dark',
+          defaultMode: "dark",
           disableSwitch: false,
           respectPrefersColorScheme: false,
-        }},
+        },
+      },
       navbar: {
-        title: 'Superflows',
+        title: "Superflows",
         logo: {
-          alt: 'Superflows Logo',
-          src: 'img/superflowsFavicon.png',
+          alt: "Superflows Logo",
+          src: "img/superflowsFavicon.png",
         },
         items: [
           {
-            type: 'docSidebar',
-            sidebarId: 'tutorialSidebar',
-            position: 'left',
-            label: 'Docs',
+            type: "docSidebar",
+            sidebarId: "tutorialSidebar",
+            position: "left",
+            label: "Docs",
           },
           {
             to: "blog",
-            position: 'left',
-            label: 'Product Roadmap',
+            position: "left",
+            label: "Product Roadmap",
           },
           {
-            href: 'https://github.com/Superflows-AI',
-            label: 'GitHub',
-            position: 'right',
+            href: "https://github.com/Superflows-AI",
+            label: "GitHub",
+            position: "right",
           },
         ],
       },
       footer: {
-        style: 'dark',
+        style: "dark",
         links: [
           {
-            title: 'Docs',
+            title: "Docs",
             items: [
               {
-                label: 'Introduction',
-                to: '/docs/intro',
+                label: "Introduction",
+                to: "/docs/intro",
               },
               {
-                label: 'Getting Started',
-                to: '/docs/getting-started/onboarding',
+                label: "Getting Started",
+                to: "/docs/getting-started/onboarding",
               },
               {
-                label: 'API',
-                to: '/docs/api',
+                label: "API",
+                to: "/docs/category/api-specification",
```

Comment:
this is an acutal change to the code

Reply:
<img width="279" alt="image" src="https://github.com/Superflows-AI/docs/assets/28537547/1cd6d793-80f8-415e-8e50-0d26d2aa5371">
`api` link here is broken